jalib/jalloc.cpp

```diff
@@ -30,6 +30,8 @@
 #include "jalloc.h"
```

Comment:
@aayushi363 ,
    Could you please test by deleting this commit, and seeing if everything still works?
    I agree with @karya0 that (a) we should not be modifying this code for _all_ CPUs -- just for riscv, and (b) I suspect that for riscv also, you won't need this code now that you've rebased to the most recent 'main' branch.  (I assume that you have rebased to the recent 'main'.  I didn't check.)